.travis.yml

```diff
@@ -23,6 +23,10 @@ matrix:
       env: MONGOID_VERSION=6
       before_script:
         - bundle exec danger
+    - rvm: 2.3.1
+      env: MONGOID_VERSION=7
+      before_script:
```

Comment:
We only need to run `danger` once in the whole build.

Reply:
OK, I will fix it. but ruby versions also need  to consider v2.4.x and v2.5.x. Thhere are some diffrences between 2.4 and 2.5.